src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -1824,7 +1751,7 @@ bool IsComInterop (IMarshalInfoProvider marshalInfoProvider, TypeReference param
 			return false;
 		}
 
-		bool AnalyzeGenericInstantiationTypeArray (in MultiValue arrayParam, ref ReflectionPatternContext reflectionContext, MethodReference calledMethod, IList<GenericParameter> genericParameters)
+		bool AnalyzeGenericInstantiationTypeArray (in AnalysisContext context, in MultiValue arrayParam, MethodReference calledMethod, IList<GenericParameter> genericParameters)
```

Comment:
Nit: Maybe we should keep naming it analysisContext to not confuse it with the linker context